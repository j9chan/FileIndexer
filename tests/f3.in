tests/book1.in