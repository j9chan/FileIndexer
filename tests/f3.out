826 the
385 of
335 a
313 and
299 to
176 he
171 in
170 you
163 that
152 his
